webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -194,7 +198,7 @@ function isMultple() {
 
 function getBulkParam() {
   return isMultple()
-    ? { search: $('#search').val() }
+    ? { search: document.getElementsByClassName('search-input')[0].value }
```

Comment:
suggestion: maybe we can get the search value from the `redux` `store` 
or maybe get it from the current `url` parameters.

Reply:
getting the search from URI now

Reply:
thanks, what about the `pagination data` and `perPage`,
could they also be achieved from `redux` or `URI` instead of the `DOM` ?

Reply:
the pagination is not connected, and those params are not required in the URI (and defaults are implemented in the server), so right now this is the best implementation.